Sources/_CryptoExtras/Util/SubjectPublicKeyInfo.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  SubjectPublicKeyInfo.swift
+//
+//
+//  Created by Gautier Delorme on 24/09/2024.
+//
```

Comment:
Can you add the license header that is present in other files and remove this header?